examples.md

```diff
@@ -62,6 +63,28 @@ steps:
       ${{ runner.os }}-go-
 ```
 
+## Haskell - Cabal
+
+We cache the elements of the Cabal store separately, as the entirety of `~/.cabal` can grow very large for projects with many dependencies.
+
+```yaml
+- uses: actions/cache@v1
+  name: Cache ~/.cabal/packages
```

Comment:
Caching this folder is a very bad idea as it'll eat up half a GiB of cache space for very little benefit; this is why I intentionally did not include this in #38 and also why `haskell-ci` avoids caching that folder for Travis CI